### ISSUE TITLE
Prism SPOT EBS volumes aren't tagged

### ISSUE THREAD

**Issue by dhananjays** [silverleafcaps/iceberg#6]
EBS root volume attached to the Prism SPOT backtest instances aren't getting tagged. These therefore get clubbed under "NoUser" category in billing report, thereby making cost analysis hard.

The EC2 instance on-the-other hand are tagged correctly. Similarly EBS volumes attached to prism host are also being tagged correctly.

**Comment by saurabhdas** [comment]
Indeed. This is a bug in AWS Batch. Will raise an issue with them and also check if we can implement a passive workaround.
